src/main/appOne.java

```diff
@@ -0,0 +1,11 @@
+class Test{
```

Comment:
line no: 3 - Consider using .equals() method for String comparison instead of == operator.